lib/discourse.rb

```diff
@@ -177,13 +177,19 @@ def self.handle_job_exception(ex, context = {}, parent_logger = nil)
 
     job = context[:job]
 
+    # mini_scheduler direct reporting
     if Hash === job
       job_class = job["class"]
       if job_class
         job_exception_stats[job_class] += 1
       end
     end
 
+    # internal reporting
+    if job.class == Class && ::Jobs::Base > job
```

Comment:
```suggestion
    if job.class == Class && job.is_a?(::Jobs::Base)
```

Just wondering if this will work as well.

Reply:
I think it works, agree it is more readable.